www/js/diary.js

```diff
@@ -2,7 +2,11 @@ import angular from 'angular';
 import LabelTab from './diary/LabelTab';
 
 angular.module('emission.main.diary',['emission.main.diary.services',
-                                      'emission.survey',
+                                      'emission.survey.external.launch',
+                                      'emission.survey.multilabel.buttons',
+                                      'emission.survey.multilabel.infscrollfilters',
+                                      'emission.survey.enketo.add-note-button',
+                                      'emission.survey.enketo.trip.infscrollfilters',
```

Comment:
I got rid of the `emission.survey` module because it didn't have anything in it anymore. But it had these dependencies.
So in the one place `emission.survey` was used, I had to copy its dependencies so the dependency tree stayed the same.